src/rtpghi~.c

```diff
@@ -193,8 +189,16 @@ void rtpghi_tilde_setup(void) {
                               CLASS_DEFAULT,
                                A_GIMME,
                               0);
-	//to do: add methods to change do_causal_pd, tol_pd on runtime
-   class_addmethod(rtpghi_tilde_class,
-                   (t_method)rtpghi_tilde_dsp, gensym("dsp"), A_CANT, 0);
-   CLASS_MAINSIGNALIN(rtpghi_tilde_class, t_rtpghi_tilde, f);
-}
+	//to do: maybe change tol_pd on runtime (while DSP turned ON)
+	class_addbang  (rtpghi_tilde_class, rtpghi_tilde_res);
+	class_addmethod(rtpghi_tilde_class,
+			(t_method)rtpghi_tilde_causal, gensym("set_c"),
+			A_DEFFLOAT, 0);
+	/*
+	class_addmethod(rtpghi_tilde_class,
+			(t_method)rtpghi_tilde_tol, gensym("set_t"),
```

Comment:
same as above.

it really makes patches easier to read if they spell `[tolerance 0.54, causal 1(` instead of `[set_t 0.54, set_c 1(`


apart from keeping yourself sane, this also follows best practice as laid out by other built in objects and extra objects (such as `sigmund~`).
i find 0 occurences of the `set_<shortname>` message pattern in the Pd core code and only *very* few occurences of the `set<longname>` pattern, presumably for good reasons.